src/components/atoms/Button/Button.tsx

```diff
@@ -94,6 +94,7 @@ const ButtonRoot = styled(ButtonForwardRef)<ButtonProps>(
         backgroundColor: theme.palette[mainColor][400],
         height: '40px',
         '&:hover': {
+          boxShadow: 'none',
```

Comment:
I believe you can move `boxShadow` to the top of so you don't have repeat `boxShadow` multiple times

Reply:
Moved to themeProps instead, I tried on top but does not work.